build/Signing/SignToolConfig.json

```diff
@@ -31,7 +31,35 @@
                 "Sdks/Microsoft.NET.Sdk/tools/net46/ru/Microsoft.NET.Build.Tasks.resources.dll",
                 "Sdks/Microsoft.NET.Sdk/tools/net46/tr/Microsoft.NET.Build.Tasks.resources.dll",
                 "Sdks/Microsoft.NET.Sdk/tools/net46/zh-Hans/Microsoft.NET.Build.Tasks.resources.dll",
-                "Sdks/Microsoft.NET.Sdk/tools/net46/zh-Hant/Microsoft.NET.Build.Tasks.resources.dll"
+                "Sdks/Microsoft.NET.Sdk/tools/net46/zh-Hant/Microsoft.NET.Build.Tasks.resources.dll",
+                "Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft.NET.Build.Extensions/tools/netcoreapp1.0/Microsoft.NET.Build.Tasks.dll",
+                "Sdks/Microsoft.NET.Build.Extensions/msbuildExtensions/Microsoft.NET.Build.Extensions/tools/netcoreapp1.0/cs/Microsoft.NET.Build.Tasks.resources.dll",
```

Comment:
Good catch on signing, but it looks like the assembly names still needs to be edited.